src/lib.rs

```diff
@@ -69,6 +69,7 @@
 //! [`Druid`]: https://docs.rs/druid
 //! [`libm`]: https://docs.rs/libm
 
+#![feature(const_fn_floating_point_arithmetic, const_trait_impl, const_ops)]
```

Comment:
note: i can remove these and make the functions non-const but someone here may have better idea